src/main/java/net/dv8tion/jda/api/JDA.java

```diff
@@ -316,6 +317,17 @@ default JDA awaitReady() throws InterruptedException
      */
     OkHttpClient getHttpClient();
 
+    /**
+     * Direct access to audio (dis-)connect requests.
+     * <br>This should not be used when normal audio operation is desired.
+     *
+     * <p>The correct way to open and close an audio connection is through the {@link Guild Guild's}
+     * {@link AudioManager}.
+     *
+     * @return The {@link DirectAudioController} for this JDA instance
+     */
+    DirectAudioController getAudioController();
```

Comment:
We should probably call this `getDirectAudioController` so as to prevent mixups with `Guild#getAudioController()`